libs/ballot-interpreter/src/hmpb-rust/image_utils.rs

```diff
@@ -293,7 +293,7 @@ pub fn detect_vertical_streaks(
 ) -> Vec<PixelPosition> {
     const PERCENT_BLACK_PIXELS_IN_STREAK: f32 = 0.75;
     const MAX_WHITE_GAP_PIXELS: PixelUnit = 15;
-    const BORDER_COLUMNS_TO_EXCLUDE: PixelUnit = 5;
+    const BORDER_COLUMNS_TO_EXCLUDE: PixelUnit = 20;
```

Comment:
I think this is a reasonable approach and am happy to move forward with it.

I wonder if we could keep this threshold lower, however, if we cropped all black columns from the edges of the image (similar to our cropping logic for the top/bottom)

Reply:
We might be able to, but in at least one example I've seen the first 4 pixels at the left edge actually do contain some white, but the next 7 are pure black. That's why that example failed with 5. We could succeed with that particular ballot with a value of 12, but only barely. It's possible that 15 would be sufficient, but I deemed the benefit of detecting streaks in that region not to be worth the risk of false positives.

However, this is not based on a lot of evidence and I'm speculating. If we think it's worth doing, I could do a more thorough analysis with the fi-7180 to see what the distribution of the black sides looks like.

Reply:
Ah ok interesting that there's some white on the edge first. Given what you've seen, a margin of 20 pixels seems reasonable.

We could also consider setting different margin values for different scanning hardware, but seems like we don't have strong evidence requiring that currently.